src/lib/output/themes/default/partials/index.tsx

```diff
@@ -67,8 +67,19 @@ export function index(context: DefaultThemeRenderContext, props: ContainerReflec
     }
 
     return (
-        <section class="tsd-panel-group tsd-index-group">
-            <section class="tsd-panel tsd-index-panel">{content}</section>
-        </section>
+        <>
+            {props instanceof DeclarationReflection &&
+                props.kind === ReflectionKind.Module &&
+                props.readme?.length !== 0 && (
+                    <section class="tsd-panel-group tsd-index-group">
```

Comment:
This should definitely not be a `tsd-index-group`, that class refers to the page index, it isn't just a generic container.

Reply:
I have removed `tsd-index-group`